src/transformers/modeling_utils.py

```diff
@@ -3588,6 +3588,14 @@ def from_pretrained(
 
             if quantization_config is None:
                 quantization_config = AwqConfig.from_dict(config.quantization_config)
+            # In case a user passes a `AwqConfig` with `do_fuse=True` for models that have
+            # a `modules_to_not_convert` attribute we need to manually set that attribute into the
+            # passed `quantization_config`
+            elif (
```

Comment:
It's not obvious how this change relates to mixtral here - either from the AWQ fuse mapping or the test. Is if it's addressing a general bug we should have a test to cover it

Reply:
It is something that has been addressed recently in https://github.com/huggingface/transformers/pull/28239 - this covers a bug where awq + fused modules does not deal properly with fused modules + `modules_to_not_convert`. I think having the mixtral and llava test (the llava test is alread there) should perhaps be already sufficient as it cover most of the usecase of `modules_to_not_convert` + fused modules. What do you think?